src/Pages/home/Components/Title.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import brainImg from "../../../assets/brain_welcome.png";
+import AddIcon from "../../../assets/add.png";
+
+//todo Style
+const useStyle = makeStyles(() =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+
+    //! Box
+    boxWelcome: {
+      width: "616px", //decrease 15%
+      height: "147px",
+      marginTop: "48px",
```

Comment:
I saw many marginTop rules(this, line 30 and line 60). If you want to set all items with the same margin or padding rules, you can set it at root(that is line 12).